### PR TITLE
DEV-3886-1270 multorsum

### PR DESCRIPTION
**Pull request by eslopfer**
Write readme for multorsum.

### CODE REVIEW

File: subjects/multorsum/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## multorsum
+
+### Instructions
+
+Write a function that receives a slice of int and and int representing the initial value. If the int is odd it should multiply the accumulated value. If it is even, it should be added instead. Return the result. If the slice is empty return 0.
```
Comment (Hamzaelkhatri) [inline subjects/multorsum/README.md:5]: > If the slice is empty return 0.

The result of your solution `MultOrSum([]int{}, -100)` ==> `-100` not `0`

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: subjects/multorsum/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eslopfer) [reply subjects/multorsum/README.md:5]: What I meant to say is `a slice of int and an int`, I'll fix the typo and keep the article if that's ok for you. Thanks for your review.

Comment (eslopfer) [reply subjects/multorsum/README.md:5]: Good catch thanks. I'll fix it.

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/multorsum/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## multorsum
+
+### Instructions
+
+Write a function that receives a slice of int and an int representing the initial value. If the int is odd it should multiply the accumulated value. If it is even, it should be added instead. Return the result. If the slice is empty return 0.
```
Comment (MSilva95) [inline subjects/multorsum/README.md:5]: Is this the same as you said? Im a bit confused with the readme...

Write a function that receives two arguments, a slice of int and an int representing the initial value.

- If the initial value is odd it should multiply the accumulated value of the slice.
- If it is even, it should be added instead.
- If the slice is empty return 0.

File: subjects/multorsum/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	fmt.Println(MultOrSum([]int{1, 2, 3, 4}, 0 ))
+	fmt.Println(MultOrSum([]int{1, -2, 3, 4}, 0 ))
+}
+```
+
+And its output :
+
+```console
+$ go run . | cat -e
+19$
```
Comment (MSilva95) [inline subjects/multorsum/README.md:32]: im not sure i understood well but isn't this supposed to be 30?

I dont understand this and the negative values can you explain a bit more?

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/multorsum/README.md (hunk 2 above)
Comment (eslopfer) [reply subjects/multorsum/README.md:5]: Thanks for pointing this out, I'll be more specific regarding the initial value. The numbers that define whether is going to be multiplied or added are the ones from the slice.

File: subjects/multorsum/README.md (hunk 3 above)
Comment (eslopfer) [reply subjects/multorsum/README.md:32]: I explained in the previous comment. Will add better description to readme.

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

File: subjects/multorsum/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+## multorsum
+
+### Instructions
+
+Write a function that receives a slice of `int` and an `int` representing the initial value as arguments. 
+
+You should traverse the slice and for each `int` check the following restrictions:
```
Comment (tiagocollot) [inline subjects/multorsum/README.md:7]: I would suggest to rephrase this  as something like: "The function should go through the slice......restrictions:"

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/multorsum/README.md (hunk 4 above)
Comment (jotapero) [reply subjects/multorsum/README.md:7]: Agreed. Fixed

**Pull request merged**